presto-main/src/test/java/com/facebook/presto/operator/aggregation/TestTypedSet.java

```diff
@@ -130,9 +134,29 @@ public void testBigintSimpleTypedSet()
         }
     }
 
+    @Test
+    public void testMemoryExceeded()
+            throws Exception
+    {
+        try {
+            TypedSet typedSet = new TypedSet(BIGINT, 10, FUNCTION_NAME);
+
+            for (int i = 0; i <= TypedSet.FOUR_MEGABYTES + 1; i++) {
+                Block block = createLongsBlock(Collections.nCopies((int) 1, (long) i));
```

Comment:
fwiw,i guess no codebase is  consistent, as doesn't import it statically
(from 2-28-17)

as long as it's clear, that method we should do static imports on